packages/forma-36-react-components/src/components/Tabs/Tabs.css

```diff
@@ -5,6 +5,7 @@
 
 .Tabs {
   display: flex;
+  padding: 0 var(--spacing-l);
```

Comment:
@m10l @joaoramos  I added some extra padding left and right to visually balance the tab panel but I'm not sure if it is needed. Any thoughts on that?